### ISSUE TITLE
aRDP: Option to not need a domain.

### ISSUE THREAD

**Issue by JoshCodeStuff** [iiordanov/remote-desktop-clients#556]
Hello! I attempted to use aRDP to connect to the built in RDP server in GNOME (on linux). aRDP keeps asking for a domain which GNOME does not provide for their RDP server. Some other clients connect without the domain, so I know the server works, but they all have horrible interfaces.

Thanks!

**Comment by iiordanov** [comment]
Hello, I assume this report is about failing to connect to Ubuntu 24.04. The issue is not that the app demands a domain. In fact, the error is not due to credentials at all.

The logs related were:

```
Jul 06 10:46:32 h gnome-remote-de[63742]: [RDP] Client did not advertise support for the Graphics Pipeline, closing connection
Jul 06 10:46:32 h gnome-remote-desktop-daemon[63742]: [10:46:32:67] [63742:0000f9b7] [ERROR][com.freerdp.core.peer] - [rdp_peer_handle_state_demand_active]: [CONNECTION_STATE_CAPABILITIES_EXCHANGE_DEMAND_ACTIVE] freerdp_peer::Capabilities() callback failed
Jul 06 10:46:32 h gnome-remote-desktop-daemon[63742]: [10:46:32:67] [63742:0000f9b7] [ERROR][com.freerdp.core.transport] - [transport_check_fds]: transport_check_fds: transport->ReceiveCallback() - STATE_RUN_FAILED [-1]
Jul 06 10:46:32 h gnome-remote-de[63742]: [RDP] Network or intentional disconnect, stopping session
```

The issue is that the app doesn't gave Gfx pipeline enabled by default. To work around this issue, please tap Advanced Settings and enable the "RDP Gfx" setting, and the connection will work. I'll make it enabled by default in a future release.

**Comment by AndKe** [comment]
![SmartSelect_20240706_184500_aRDP Free](https://github.com/iiordanov/remote-desktop-clients/assets/1468842/649f5f59-8df9-455c-8a4e-568841689600)
![SmartSelect_20240706_184420_aRDP Free](https://github.com/iiordanov/remote-desktop-clients/assets/1468842/f656a423-cd03-4044-a872-81fd61efe57c)

I have enabled the RDP gfx setting, and my situation is seen in the screenshots above.

Please note that before going on vacation, I tested successfully Rammina for Ubuntu RDP, it worked fine. Now , my laptop won't connect when abroad, (using WireGuard) I have good network connection home, can SSH the server at home, but not the workstation above, I also forgot to install SSH server on it after reinstalling it for the first time since Ubuntu 16.04 ...
TLDR my issues right now may be on the RDP server side. I do know that it won't sleep and it is running, but cannot SSH to it, and suspect that display sleep may be the culprit.

**Comment by iiordanov** [comment]
The app now has Gfx enabled by default and connections to Ubuntu should work fine. please let me know if that's not the case and i can reopen.
